base/ipfilter.go

```diff
@@ -0,0 +1,174 @@
+package base
+
+import (
+	"bytes"
+	"github.com/fsnotify/fsnotify"
+	"github.com/spf13/viper"
+	"net"
+	"net/http"
+	"regexp"
+	"strings"
+)
+
+type Prohibited struct {
+	ips     map[string]bool
+	subnets []*subnet
+	ranges  []*iprange
+}
+
+type subnet struct {
+	ip    string
+	ipnet *net.IPNet
+}
+
+type iprange struct {
+	start net.IP
+	end   net.IP
+}
+
+var ProhibitedIPs Prohibited
+var ProhibitedDomains string
+
+func InitProhibitedIPs() {
+	ProhibitedIPs.ips = make(map[string]bool)
+	config := viper.New()
+	config.SetConfigName("prohibited")
+	config.SetConfigType("yaml")
+	config.AddConfigPath("/usr/local/production/config/")
+
+	err := config.ReadInConfig()
+	if err != nil {
+		Zlog.Errorf("Falied to Initialise the Prohibited domain data: %s", err.Error())
```

Comment:
Got a typo to Falied -> Failed. Why putting capital letter to Initialize and Prohibited ?

Reply:
@susmithp bits we can pick up through spell-check -- will suggest plugins offline --

-- corrected versions for you for ipfilter.go --

Zlog.Errorf("Failed to initialize prohibited domains list: %s", err.Error())
Zlog.Infof("Config file changed")
Zlog.Infof("User email domain [%s] is safe to procced", userDomain)

Reply:
@arunkoshy I have corrected the typo, please re-review.